### PR TITLE
Update Tools Nuget package descriptions

### LINKED ISSUE #9439: Improve Tools packages descriptions
See https://stackoverflow.com/questions/45703613/whats-the-difference-between-efcore-tools-and-efcore-tools-dotnet

### PR DESCRIPTION
**Pull request by ErikEJ**
fixes #9439

-verified by building and inspecting generated packages with Nuget Package Explorer

### CODE REVIEW

File: src/EFCore.Tools.DotNet/EFCore.Tools.DotNet.nuspec [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <authors>Microsoft</authors>
     <copyright>Copyright © Microsoft Corporation</copyright>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>Entity Framework Core .NET Command Line Tools. Includes dotnet-ef.</description>
+    <description>Entity Framework Core Tools for the “dotnet” Command Line Interface. Includes the “dotnet ef migrations”, “dotnet ef dbcontext”, and “dotnet ef database” commands.</description>
```
Comment (bricelam) [inline src/EFCore.Tools.DotNet/EFCore.Tools.DotNet.nuspec]: I don't think we want to use smart quotes here.

Comment (ErikEJ) [reply src/EFCore.Tools.DotNet/EFCore.Tools.DotNet.nuspec]: Fixed

**Comment by ErikEJ** [comment]
Ping?

**Comment by ErikEJ** [comment]
Pong?

**Comment by divega** [comment]
@ErikEJ finally got to this. I rebased, and moved your new descriptions to the csproj files.

@bricelam if this looks good to you, feel free to merge.

**Comment by divega** [comment]
@bricelam new version inspired by https://github.com/aspnet/Microsoft.Data.Sqlite/blob/dev/src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj. 

What do you think?

**Comment by divega** [comment]
Thanks @ErikEJ!

**Pull request merged**